api/proto/config/storage.proto

```diff
@@ -34,13 +35,17 @@ message DatastoreConfig {
 
 	string apiKey = 3;
```

Comment:
Is this also part of the cryptoblock? Please add comment whether or not, and which field in the cryptoblock (I'd assume "identity")

Reply:
Agree, will do.